emp/migrate_to_io2.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import boto3
+import math
+import logging
+import time
+import botocore
+import threading
+
+AWS_REGION_ID=''
+AWS_ACCESS_KEY_ID=''
+AWS_SECRET_ACCESS_KEY=''
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='[%(asctime)s] %(levelname)s - %(message)s',
+)
+logger = logging.getLogger('PF9')
+
+class MigrateVolume:
+    def __init__(self):
+        self.ec2_client = boto3.client('ec2', aws_access_key_id=AWS_ACCESS_KEY_ID, aws_secret_access_key=AWS_SECRET_ACCESS_KEY, region_name=AWS_REGION_ID)
+        self.polltime = 30
+
+    def calculate_max_iops(self, volume_size):
+        max_iops = min(math.ceil(volume_size * 500), 64000)
```

Comment:
extract 500 to default iops per GB. and get that as flag (not required value)

Reply:
Done